data_resource_api/factories/orm_factory.py

```diff
@@ -65,43 +65,74 @@ def evaluate_foreign_key(self, foreign_keys, field_name, field_type):
                 return True, foreign_key_reference
         return False, None
 
-    def create_sqlalchemy_fields(self, fields: dict, primary_key, foreign_keys=[]):
+    def create_sqlalchemy_fields(self, descriptor_fields: list(dict), primary_key: str, foreign_keys=[]):
```

Comment:
@loganripplinger - the new annotation type is raising an error. 

```
data-resource-api_1   |     def create_sqlalchemy_fields(self, descriptor_fields: list(dict), primary_key: str, foreign_keys=[]):
data-resource-api_1   | TypeError: 'type' object is not iterable
```

Can you find the correct way to do this for python 3.7.3 (i.e. the version of python speceified in the Dockerfile)? 

https://docs.python.org/3/library/typing.html

Reply:
Once you fix this, I'll finish my review.